Sources/Build/BuildDelegate.swift

```diff
@@ -396,6 +396,7 @@ public final class BuildDelegate: BuildSystemDelegate, SwiftCompilerOutputParser
 
     public func hadCommandFailure() {
         onCommmandFailure?()
+        print("\n\n * Build Failed!")
```

Comment:
It would be great to find a way to funnel the actual printing so it happens in one place, though I see that there are other places in this code where it prints directly.  This is partly because the code might want to conditionally emit ANSI formatting codes.  Could this be added to `latestFinishedText`, as in the success case?

Reply:
I think this might take some refactoring of BuildDelegate

Reply:
After conversation with @abertelrud we are going to take care of the Build Failed message later on